debugd/internal/cdbg/cmd/deploy.go

```diff
@@ -14,7 +14,9 @@ import (
 	"net"
 	"strconv"
 	"strings"
+	"time"
 
+	"github.com/avast/retry-go/v4"
```

Comment:
Please use `internal/retry/retry.go` as it should give you the same functionality